frontend/apps/app/components/SessionDetailPage/components/Chat/components/TimelineItem/components/VersionMessage/VersionMessage.tsx

```diff
@@ -74,30 +82,30 @@ const parsePatchOperations = (
   })
 }
 
-type BuildingSchemaVersion = Pick<
-  Tables<'building_schema_versions'>,
-  'patch' | 'number' | 'id'
->
-
 type Props = {
   buildingSchemaVersionId: string
-  onView?: () => void
+  onView?: (version: Version) => void
 }
 
 export const VersionMessage: FC<Props> = ({
   buildingSchemaVersionId,
   onView,
 }) => {
-  const [version, setVersion] = useState<BuildingSchemaVersion | null>(null)
+  const [version, setVersion] = useState<Version | null>(null)
   const [isPending, startTransition] = useTransition()
   const [isExpanded, setIsExpanded] = useState(false)
 
+  const handleClick = useCallback(() => {
+    if (!version) return
+    onView?.(version)
+  }, [version, onView])
+
   useEffect(() => {
     startTransition(async () => {
       const supabase = createClient()
       const { data, error } = await supabase
         .from('building_schema_versions')
-        .select('id, number, patch')
+        .select('id, number, patch, reverse_patch, building_schema_id')
```

Comment:
[ask]
Are there any plans to use `reverse_patch` and `building_schema_id` fields in the　VersionMessage component?